diff --git a/aosp-mirror/platform_bionic/libc/include/sys/system_properties.h b/aosp-mirror/platform_bionic/libc/include/sys/system_properties.h
--- a/aosp-mirror/platform_bionic/libc/include/sys/system_properties.h
+++ b/aosp-mirror/platform_bionic/libc/include/sys/system_properties.h
@@ -30,7 +30,7 @@
 #define _INCLUDE_SYS_SYSTEM_PROPERTIES_H

 #include <sys/cdefs.h>
-#include <stdbool.h>
+#define bool _Bool
 #include <stddef.h>
 #include <stdint.h>
