diff --git a/zircon/system/public/zircon/features.h b/zircon/system/public/zircon/features.h
--- a/zircon/system/public/zircon/features.h
+++ b/zircon/system/public/zircon/features.h
@@ -8,17 +8,17 @@
 // clang-format off

 // types of features that can be retrieved via |zx_system_get_features|
-#define ZX_FEATURE_KIND_CPU                   ((uint32_t)0)
-#define ZX_FEATURE_KIND_HW_BREAKPOINT_COUNT   ((uint32_t)1)
-#define ZX_FEATURE_KIND_HW_WATCHPOINT_COUNT   ((uint32_t)2)
-#define ZX_FEATURE_KIND_ADDRESS_TAGGING       ((uint32_t)3)
-#define ZX_FEATURE_KIND_VM                    ((uint32_t)4)
+#define ZX_FEATURE_KIND_CPU                   0
+#define ZX_FEATURE_KIND_HW_BREAKPOINT_COUNT   1
+#define ZX_FEATURE_KIND_HW_WATCHPOINT_COUNT   2
+#define ZX_FEATURE_KIND_ADDRESS_TAGGING       3
+#define ZX_FEATURE_KIND_VM                    4

 // arch-independent CPU features
-#define ZX_HAS_CPU_FEATURES            ((uint32_t)(1u << 0))
+#define ZX_HAS_CPU_FEATURES            (1u << 0)

 // Virtual memory features.
-#define ZX_VM_FEATURE_CAN_MAP_XOM ((uint32_t)(1u << 0))
+#define ZX_VM_FEATURE_CAN_MAP_XOM (1u << 0)

 #if defined(__x86_64__)

@@ -28,30 +28,30 @@
 #elif defined(__aarch64__)

 // arm64 CPU features
-#define ZX_ARM64_FEATURE_ISA_FP        ((uint32_t)(1u << 1))
-#define ZX_ARM64_FEATURE_ISA_ASIMD     ((uint32_t)(1u << 2))
-#define ZX_ARM64_FEATURE_ISA_AES       ((uint32_t)(1u << 3))
-#define ZX_ARM64_FEATURE_ISA_PMULL     ((uint32_t)(1u << 4))
-#define ZX_ARM64_FEATURE_ISA_SHA1      ((uint32_t)(1u << 5))
-#define ZX_ARM64_FEATURE_ISA_SHA256    ((uint32_t)(1u << 6))
-#define ZX_ARM64_FEATURE_ISA_CRC32     ((uint32_t)(1u << 7))
-#define ZX_ARM64_FEATURE_ISA_ATOMICS   ((uint32_t)(1u << 8))
-#define ZX_ARM64_FEATURE_ISA_RDM       ((uint32_t)(1u << 9))
-#define ZX_ARM64_FEATURE_ISA_SHA3      ((uint32_t)(1u << 10))
-#define ZX_ARM64_FEATURE_ISA_SM3       ((uint32_t)(1u << 11))
-#define ZX_ARM64_FEATURE_ISA_SM4       ((uint32_t)(1u << 12))
-#define ZX_ARM64_FEATURE_ISA_DP        ((uint32_t)(1u << 13))
-#define ZX_ARM64_FEATURE_ISA_DPB       ((uint32_t)(1u << 14))
-#define ZX_ARM64_FEATURE_ISA_FHM       ((uint32_t)(1u << 15))
-#define ZX_ARM64_FEATURE_ISA_TS        ((uint32_t)(1u << 16))
-#define ZX_ARM64_FEATURE_ISA_RNDR      ((uint32_t)(1u << 17))
-#define ZX_ARM64_FEATURE_ISA_SHA512    ((uint32_t)(1u << 18))
-#define ZX_ARM64_FEATURE_ISA_I8MM      ((uint32_t)(1u << 19))
-#define ZX_ARM64_FEATURE_ISA_SVE       ((uint32_t)(1u << 20))
+#define ZX_ARM64_FEATURE_ISA_FP        (1u << 1)
+#define ZX_ARM64_FEATURE_ISA_ASIMD     (1u << 2)
+#define ZX_ARM64_FEATURE_ISA_AES       (1u << 3)
+#define ZX_ARM64_FEATURE_ISA_PMULL     (1u << 4)
+#define ZX_ARM64_FEATURE_ISA_SHA1      (1u << 5)
+#define ZX_ARM64_FEATURE_ISA_SHA256    (1u << 6)
+#define ZX_ARM64_FEATURE_ISA_CRC32     (1u << 7)
+#define ZX_ARM64_FEATURE_ISA_ATOMICS   (1u << 8)
+#define ZX_ARM64_FEATURE_ISA_RDM       (1u << 9)
+#define ZX_ARM64_FEATURE_ISA_SHA3      (1u << 10)
+#define ZX_ARM64_FEATURE_ISA_SM3       (1u << 11)
+#define ZX_ARM64_FEATURE_ISA_SM4       (1u << 12)
+#define ZX_ARM64_FEATURE_ISA_DP        (1u << 13)
+#define ZX_ARM64_FEATURE_ISA_DPB       (1u << 14)
+#define ZX_ARM64_FEATURE_ISA_FHM       (1u << 15)
+#define ZX_ARM64_FEATURE_ISA_TS        (1u << 16)
+#define ZX_ARM64_FEATURE_ISA_RNDR      (1u << 17)
+#define ZX_ARM64_FEATURE_ISA_SHA512    (1u << 18)
+#define ZX_ARM64_FEATURE_ISA_I8MM      (1u << 19)
+#define ZX_ARM64_FEATURE_ISA_SVE       (1u << 20)

 // This is an obsolete name for the same thing.
 #define ZX_ARM64_FEATURE_ISA_SHA2      ZX_ARM64_FEATURE_ISA_SHA256

 // arm64 address-tagging features
-#define ZX_ARM64_FEATURE_ADDRESS_TAGGING_TBI  ((uint32_t)(1u << 0))
+#define ZX_ARM64_FEATURE_ADDRESS_TAGGING_TBI  (1u << 0)

 #elif defined(__riscv)

diff --git a/zircon/system/public/zircon/types.h b/zircon/system/public/zirc
on/types.h
--- a/zircon/system/public/zircon/types.h
+++ b/zircon/system/public/zircon/types.h
@@ -5,8 +5,8 @@
 #ifndef SYSROOT_ZIRCON_TYPES_H_
 #define SYSROOT_ZIRCON_TYPES_H_

-#include <stdbool.h>
-#include <stddef.h>
+// #include <stdbool.h>
+// #include <stddef.h>
 #include <stdint.h>
 #include <zircon/compiler.h>
 #include <zircon/errors.h>
@@ -23,7 +23,7 @@
 // A better solution would be to use <stdatomic.h> and C11 atomic operation
 // even in the kernel, but that would require modifying all the code that uses
 // the existing homegrown atomics.
-#include <stdatomic.h>
+// #include <stdatomic.h>
 #endif
 #endif

@@ -563,7 +563,7 @@ typedef int zx_futex_t;
 #ifdef _KERNEL
 typedef int zx_futex_t;
 #else
-typedef atomic_int zx_futex_t;
+// typedef atomic_int zx_futex_t;
 #endif
 #endif
 typedef int zx_futex_storage_t;
