diff --git a/zircon/system/public/zircon/features.h b/zircon/system/public/zircon/features.h
--- a/zircon/system/public/zircon/features.h
+++ b/zircon/system/public/zircon/features.h
@@ -8,7 +8,7 @@
 // clang-format off

 // types of features that can be retrieved via |zx_system_get_features|
-#define ZX_FEATURE_KIND_CPU                   ((uint32_t)0)
+#define ZX_FEATURE_KIND_CPU                   0
 #define ZX_FEATURE_KIND_HW_BREAKPOINT_COUNT   ((uint32_t)1)
 #define ZX_FEATURE_KIND_HW_WATCHPOINT_COUNT   ((uint32_t)2)
 #define ZX_FEATURE_KIND_ADDRESS_TAGGING       ((uint32_t)3)
@@ -35,7 +35,7 @@
 #define ZX_ARM64_FEATURE_ISA_SHA1      ((uint32_t)(1u << 5))
 #define ZX_ARM64_FEATURE_ISA_SHA256    ((uint32_t)(1u << 6))
 #define ZX_ARM64_FEATURE_ISA_CRC32     ((uint32_t)(1u << 7))
-#define ZX_ARM64_FEATURE_ISA_ATOMICS   ((uint32_t)(1u << 8))
+#define ZX_ARM64_FEATURE_ISA_ATOMICS   (1u << 8)
 #define ZX_ARM64_FEATURE_ISA_RDM       ((uint32_t)(1u << 9))
 #define ZX_ARM64_FEATURE_ISA_SHA3      ((uint32_t)(1u << 10))
 #define ZX_ARM64_FEATURE_ISA_SM3       ((uint32_t)(1u << 11))
